addons/rose/package.json

```diff
@@ -34,7 +34,7 @@
   },
   "dependencies": {
     "@embroider/test-setup": "^0.48.1",
-    "@hashicorp/design-system-components": "^1.3.2",
+    "@hashicorp/design-system-components": "^1.5.1",
```

Comment:
Thanks for this!!

Reply:
Yup I had to update this for the new HDS table component!